EventListener/TwitterChannelConfigSubscriber.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+/*
+ * This file is part of the Integrated package.
+ *
+ * (c) e-Active B.V. <integrated@e-active.nl>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Integrated\Bundle\SocialBundle\EventListener;
+
+use Integrated\Bundle\ChannelBundle\Event\FormConfigEvent;
+use Integrated\Bundle\ChannelBundle\Event\GetResponseConfigEvent;
+use Integrated\Bundle\ChannelBundle\IntegratedChannelEvents;
+use Integrated\Bundle\SocialBundle\Connector\TwitterAdapter;
+use Integrated\Bundle\SocialBundle\Factory\TwitterFactory;
+
+use Integrated\Common\Channel\Connector\Config\ConfigManagerInterface;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+
+/**
+ * This event listener will request a twitter access token for the twitter connector.
+ *
+ * @author Jan Sanne Mulder <jansanne@e-active.nl>
+ */
+class TwitterChannelConfigSubscriber implements EventSubscriberInterface
+{
+    /**
+     * @var TwitterFactory
+     */
+    private $factory;
+
+    /**
+     * @var ConfigManagerInterface
+     */
+    private $manager;
+
+    /**
+     * @var UrlGeneratorInterface
+     */
+    private $generator;
+
+    /**
+     * @param TwitterFactory         $factory
+     * @param ConfigManagerInterface $manager
+     * @param UrlGeneratorInterface  $generator
+     */
+    public function __construct(TwitterFactory $factory, ConfigManagerInterface $manager, UrlGeneratorInterface $generator)
```

Comment:
This makes Jenkins fails: http://jenkins.e-active.nl:8080/job/integrated-social-bundle/10/console